A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Misc.xml

```diff
@@ -294,4 +294,56 @@
             <TOOLTIPID>%((A:APU PCT RPM, Percent) 0 &gt; (A:APU PCT RPM, Percent) #AVAIL_PERCENTAGE# &lt; and)%{if}TT:COCKPIT.TOOLTIPS.UNAVAILABLE%{end} %((A:APU PCT RPM, Percent) #AVAIL_PERCENTAGE# &gt;= (A:APU SWITCH, Bool) and)%{if}TT:COCKPIT.TOOLTIPS.APU_STARTER_TURN_OFF%{else}TT:COCKPIT.TOOLTIPS.APU_STARTER_TURN_ON%{end}</TOOLTIPID>
         </UseTemplate>
     </Template>
+        <Template Name="FBW_ENGINE_Switch_Master_SubTemplate">
```

Comment:
In this case, copying only the subtemplate is fine, since the "main" template isn't doing much. But you should remove the "sub" from the name. Also, the indentation of the template is a bit wrong.